pages/jobs.tsx

```diff
@@ -1,35 +1,215 @@
 import type { NextPage } from 'next';
 import type { JobSummary } from '../lib/peopleHR';
+import type { Jobs } from 'types/pages/Jobs';
 import { getAllJobSummaries } from './api/jobs/summaries';
 import styles from 'styles/Jobs.module.scss';
 import Link from 'next/link';
+import Title from 'components/Title';
+import Layout from 'components/Layout';
+import { getJobListingPage } from 'lib/api';
+import RichText from 'components/RichText/RichText';
+import { SwishButton } from 'components/Button/Button';
+import { WomanLeanIn } from 'components/Icons/WomanLeanIn';
+import { LocationIcon } from 'components/Icons/LocationIcon';
+import { useState } from 'react';
+import Select from 'components/Select';
+
+type JobEntryProps = {
+    job: JobSummary;
+};
+
+const JobEntry = ({ job }: JobEntryProps) => (
+    <>
+        <Link href={'/jobs/' + job.slug}>
+            <a className={styles.jobTitle}>{job.title}</a>
+        </Link>
+        <p className={styles.jobDescription}>{job.description}</p>
+        <div className={styles.jobMetadataItems}>
+            <div className={styles.jobMetadata}>
+                <LocationIcon
+                    width={22}
+                    height={23}
+                    className={styles.jobMetadataIcon}
+                />{' '}
+                {job.department}
+            </div>
+            <div className={styles.jobMetadata}>
+                <LocationIcon
+                    width={22}
+                    height={23}
+                    className={styles.jobMetadataIcon}
+                />{' '}
+                <div>{job.city}</div>
+            </div>
+        </div>
+    </>
+);
+
+type CTAProps = {
+    title: string;
+    children: React.ReactNode;
+};
+
+const CTA = ({ title, children }: CTAProps) => (
```

Comment:
**Nickpick** Rename to JobCTA

Reply:
Renamed - note that this component isn't exported and is local to this file, so I think this shouldn't matter? Nicer to be more specific though